tests/Integration/Schema/Directives/HasManyDirectiveTest.php

```diff
@@ -146,7 +146,7 @@ public function testQueryHasManyWithConditionInDifferentAliases(): void
         }
 
         type Query {
-            users: [User!]! @all
+            users: [User!]! @all @orderBy(column: "id")
```

Comment:
Why did you add this?

Reply:
Simple queries like `select * from users` may be returned in the order they are stored on disk, in primary key order, or others. When we select specific columns, it is more unpredictable without using `orderBy`.

https://dba.stackexchange.com/a/6053

After some commits, I removed it and made the primary key as the default order.